### DISCUSSION TITLE
Heat Exchanger

### DISCUSSION THREAD

**Discussion by jfulem** [Hopsan/hopsan#2235] [Q&A]
Please, is it possible to model the oil heat exchanger in the hydraulic circuit? I'm very new to Hopsan.

**Comment by robbr48** [comment]
The default library in Hopsan does not yet support heat transfer and temperature. I think it could be possible to create your own component library for this though. This is of course assuming that you need to simulate the actual heat transfer, if you only need the pressures and flows you could probably simulate it using an orifice or some other resistive component.

**Reply by jfulem** [reply]
Thanks. Yes, I would like to model the heat transfer as well. I'm experienced with Modelica, but this TML approach is quite new to me.
